src/test/scala/info/bethard/timenorm/formal/ParseAnaforaXMLs.scala

```diff
@@ -9,15 +10,47 @@ object ParseAnaforaXMLs {
   // annotations that don't represent normalizable time expressions
   val skip = Set("NotNormalizable", "Frequency", "PreAnnotation")
 
+  def parseDCT(dctString: String): Seq[Int] = {
```

Comment:
I see three instances of `def parseDCT`. There should be only one, and it probably belongs in TimeMLScorer.